packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -765,6 +763,10 @@ class Viewport implements IViewport {
     // compute the radius of the enclosing sphere
     radius = Math.sqrt(radius) * 0.5;
 
+    // For 3D viewport, we should increase the radius to make sure the whole
+    // volume is visible and we don't get clipping artifacts.
+    radius = this.type === ViewportType.VOLUME_3D ? radius * 10 : radius;
```

Comment:
Is there a performance impact of this?  Is there a smaller value that would still fix things, but have less impact?

Reply:
There should not be any performance impact